app/views/catalog/_new_search.html.erb

```diff
@@ -0,0 +1,91 @@
+<%# locals: (content_classes: document_index_view_type == :gallery ? 'col-md-8 col-xxl-9 px-xxl-4 ps-lg-4 ps-2' : 'col-md-8 col-xxl-6 px-xxl-4 ps-lg-4 ps-2', hide_constraints: false, hide_facets: false) %>
+
+<h1 class="visually-hidden">New search</h1>
+
+<div class="row align-items-start mt-4">
+  <div class="sidebar col-12 col-md-4 col-xxl-3 mb-3" data-controller="analytics" data-action="hide.bs.collapse->analytics#trackFacetHide show.bs.collapse->analytics#trackFacetShow">
+    <button class="btn btn-primary d-inline-block d-md-none me-2" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-facets" aria-controls="offcanvas-facets">
+      <i class="bi bi-sliders" aria-hidden="true"></i> Filters
+    </button>
+
+    <div id="offcanvas-facets" class="offcanvas-md offcanvas-bottom offcanvas-sm-start">
+      <div class="offcanvas-body flex-column">
+        <%= render 'facet_header' %>
+        <%= render Facets::GroupComponent.new(id: 'top-filters', title: 'Top filters',
+                                              container_classes: 'facets top-filters sidenav mb-md-4',
+                                              header_classes: 'facets-heading',
+                                              body_classes: 'accordion') do |component| %>
+          <% component.with_body do %>
+            <%= render Blacklight::FacetComponent.with_collection(blacklight_config.facet_fields_in_group('top'), response: @response) %>
+          <% end %>
+        <% end %>
+
+        <%= render Facets::GroupComponent.new(id: 'other-filters', title: 'Other filters',
+                                              show_button: true,
+                                              container_classes: 'facets other-filters sidenav mb-md-4',
+                                              header_classes: 'facets-heading',
+                                              body_classes: 'accordion') do |component| %>
+          <% component.with_body do %>
+            <div data-facet-list-target="hideFacets" id="hidden-facets">
+              <%= render Blacklight::FacetComponent.with_collection(blacklight_config.facet_fields_in_group(nil).sort_by(&:label), response: @response) %>
+            </div>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+  <div class="col-lg-6 col-xl-8"><section class="px-lg-3 py-1 tips">
+      <header class="d-flex flex-wrap align-items-center h3"><span class="text-nowrap">Search tips</span>
```

Comment:
This content is almost the same but different from the home page search tips. I'm not sure trying to be clever and reconciling them is worth the added complexity, but could be done if we really cared 🤷‍♂️